release-common-lib/src/main/java/org/reactome/release/common/database/InstanceEditUtils.java

```diff
@@ -70,18 +74,25 @@ public static GKInstance createDefaultIE(MySQLAdaptor dba, Long defaultPersonId,
 			{
 				dba.storeInstance(newIE);
 			}
-//			else
-//			{
-//				logger.info("needStore set to false");
-//			}
+			//else
```

Comment:
Remove dead code?

Reply:
Sure, I can.  I didn't change any of the code in the PR as a whole.  I just added JavaDocs, updated the pom.xml, and did some formatting.

Reply:
Okay, done!